src/admin/python/lsst/qserv/admin/qservCli/launch.py

```diff
@@ -1067,15 +1070,25 @@ def integration_test(
         args.extend(bind_args(qserv_root=qserv_root, bind_names=bind))
     add_network_option(args, project)
     args.extend([qserv_image, "entrypoint", "--log-level", "DEBUG", "integration-test"])
+
     for opt, var in (
+        ("--prep", prep),
         ("--unload", unload),
         ("--reload", reload),
-        ("--run-tests", run_tests),
-        ("--compare-results", compare_results),
     ):
         if var:
             args.append(opt)
 
+    for opt, var in (
+        ("run-tests", run_tests),
+        ("compare-results", compare_results),
+    ):
+        if var:
+            prefix = "--"
+        else:
+            prefix = "--no-"
+            args.append(f"{prefix}{opt}")
+
```

Comment:
this is cute but for clarity I think this is better
```
args.append("--run-tests" if run_tests else "--no-run-tests")
args.append("--compare-results" if run_tests else "--compare-results")
```

Reply:
:+1: